lib/eventbrite/finder.rb

```diff
@@ -0,0 +1,33 @@
+require "httparty"
+
+module Eventbrite
+  class Finder
+    include ::HTTParty
```

Comment:
I'm going try it without the `::` again, but tests were complaining about not being able to find it under the `Eventbrite` scope.